acceptance/tests/provider/package/dpkg_ensure_held_package_is_latest.rb

```diff
@@ -0,0 +1,26 @@
+test_name "puppet provider package dpkg"
```

Comment:
Test name should be in relation with the filename and should define what is the scope of the test
And looking at the following file you have the sane `test_name`